### PR TITLE
Update repo readme: overview and description

### PR DESCRIPTION
**Pull request by slemeur**
Signed-off-by: Stévan Le Meur <1636769+slemeur@users.noreply.github.com>

### What does this PR do?

Some minor updates on the readme of the repository on the description and the overview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,31 @@ We are seeking users who are interested in helping us improve Podman Desktop.
...
 ## Overview
-This tool allows to browse, manage lifecycle of containers, inspect containers, images from different container engines.
 
-It includes a tray icon support.
- 
-It can connect to multiple engines at the same time and provides an unified interface.
+Podman Desktop enables you to easily work with containers and kubernetes from your local environment. Podman Desktop leverages Podman Engine to provide a lightweight and daemon-less container tool.
 
-Specific runtime plug-ins provides support for different container engine.
+**Podman and Kubernetes/OpenShift Local**
```
Comment (benoitf) [inline README.md:28]: there is no OpenShift Local

File: README.md [hunk 2]
```diff
@@ -20,15 +20,31 @@ We are seeking users who are interested in helping us improve Podman Desktop.
 Register your interest to join our Early Adopter Program by filling <a href="https://forms.gle/ow73dV7Ce3YLzoXH7" target="_blank">this form</a>.
 
 ## Overview
-This tool allows to browse, manage lifecycle of containers, inspect containers, images from different container engines.
 
-It includes a tray icon support.
```
Comment (benoitf) [inline README.md:25]: Tray icon is missing

**Review by benoitf — COMMENTED** [review]
IMHO the wording is looking more 'marketing materials' than what usually does a project README

File: README.md (hunk 2 above)
Comment (slemeur) [reply README.md:25]: added

**Review by benoitf — APPROVED** [review]

**Pull request merged**